deployment/terraform/README.md

```diff
@@ -0,0 +1,48 @@
+# Description
+
+With this part of kuksa.cloud project it is possible to deploy a Keycloak server to Azure cloud platform secured with a 
+valid Let's Encrypt certificate without any manual steps. Please keep in mind that there is no claim to use this deployment
+in a productive setup.
+
+## Used Technologies  
+- Terraform
+- Azure
+- AKS
+- Helm
+- Let’s Encrypt
+- Keycloak
+
+# How-to use
+
+```sh
+# clone kuksa.cloud repository
+git clone https://github.com/eclipse/kuksa.cloud.git
+cd kuksa.cloud/deployment/terraform
+
+# login to Azure and set subscription
+az login
+az account set --subscription <ID>
+
+# prepare infrastructure configuration
+cp ./terraform.tfvars.template ./terraform.tfvars
+code ./terraform.tfvars
+
+# deploy infrastructure
+source ./prepare_environment.sh
+terraform workspace select <dev, stage or prod>
+terraform init -upgrade
+terraform plan
```

Comment:
apply prints a plan anyway

Reply:
For current versions of Terraform this is correct. This how-to should be independent of Terraform version but raise the claim to list all steps you should do even if you use "-auto-approve" flag regularly or in automation.